ts/private/ts_project.bzl

```diff
@@ -86,18 +86,23 @@ See https://github.com/aspect-build/rules_ts/issues/228 for more details.
     arguments.add_all(ctx.attr.args)
 
     outdir = _lib.join(ctx.label.package, ctx.attr.out_dir) if ctx.attr.out_dir else ctx.label.package
+    if ctx.label.workspace_root:
```

Comment:
I think we can drop the `if` and just do this 100% of the time?

Reply:
That would also cover the `outdir == "":` case, so we can just have a single `outdir = _lib.join(..)`?